README.md

```diff
@@ -1,4 +1,8 @@
 # ILGPU
+![CI](https://github.com/G-Research/consuldotnet/workflows/CI/badge.svg?style=flat)
```

Comment:
I think it would be better to import a CI-badge instead of referring to another repository for this purpose. What do you think?

Reply:
heh, happens to the best of us 😅